### DISCUSSION TITLE
It is possible to add subgroups to favorites.

### DISCUSSION THREAD

**Discussion by henriquelucas** [rustdesk/rustdesk#4130] [General]
### Description

It is possible to add subgroups to favorites.

For example, I run two companies with Rustdesk and Fca all my favorites mixed together. Even with a search field, perhaps this organization by group would be better.

### Impact

A better organization of devices saved in Favorites, like h´pa in Teamviews for example.

**Comment by henriquelucas** [comment]
Can I expect something about this suggestion? @rustdesk

**Comment by dinger1986** [comment]
Can use tags with pro in the addressbook
